common/script/ops/openMysteryItem.js

```diff
@@ -28,17 +27,13 @@ module.exports = function openMysteryItem (user, req = {}, analytics) {
     });
   }
 
-  if (typeof window !== 'undefined') {
-    if (!user._tmp) user._tmp = {};
-    user._tmp.drop = item;
-  }
-
   if (req.v2 === true) {
     return user.items.gear.owned;
   } else {
     return [
       user.items.gear.owned,
       i18n.t('mysteryItemOpened', req.language),
+      item,
```

Comment:
I'm pretty sure that the res.respond code only works with 3 arguments (statusCode, data, message) https://github.com/HabitRPG/habitrpg/blob/develop/website/server/middlewares/api-v3/response.js#L3 and yeah it would probably break API consumers. @crookedneighbor what do you think?